.install/common_installations.sh

```diff
@@ -3,28 +3,14 @@ set -e
 OS_TYPE=$(uname -s)
 MODE=$1 # whether to install using sudo or not
 
-activate_venv() {
-	echo "copy ativation script to shell config"
-	if [[ $OS_TYPE == Darwin ]]; then
-		echo "source venv/bin/activate" >> ~/.bashrc
-		echo "source venv/bin/activate" >> ~/.zshrc
-	else
-		echo "source $PWD/venv/bin/activate" >> ~/.bash_profile
-	fi
-}
+pip3 install --upgrade pip
+pip3 install -q --upgrade setuptools
+echo "pip version: $(pip3 --version)"
+echo "pip path: $(which pip3)"
 
-python3 -m venv venv
-activate_venv
-source venv/bin/activate
-
-pip install --upgrade pip
-pip install -q --upgrade setuptools
-echo "pip version: $(pip --version)"
-echo "pip path: $(which pip)"
-
-pip install -q -r tests/pytest/requirements.txt
+pip3 install -q -r tests/pytest/requirements.txt
 # These packages are needed to build the package
-pip install -q -r .install/build_package_requirements.txt
```

Comment:
why did it change? if we are not using build_package_requirements so let's delete it, if we are replace with it

Reply:
unsure. i think this was a result of a weird merge from master. i don't remember this file showing up as a conflict